validation/report.md

```diff
@@ -134,6 +162,18 @@ Figure 7
 Although the results for the weight percentage for the city of Sao Paulo is at 83.68%, there is a high discrepancy between OSM mean and Official mean results. The 16% difference is driven in part by the total OSM destination count (1562 points) and the Official destination count (34 points). As a robustness test, the points originally taken out of the official data were put back in to the validation analysis. This addition of approximately 900 destinations does not significantly change indicator values. In both scenarios, the true weight value indicates that both OSM and Official datasets contain similar information, either destination points are present within a hexagon boundary or no destination points are present.
 
 
+## External Validation
+
+### Edge
+Google Maps Satellite View was used to understand what exists in spaces that contains Official edges, but not OSM edges. For each city, one example was examined. In all three examples, the edges should not be included in the analysis, as they do not represent paths that are a part of network circulation. 
+ 
+In Bern, the space examined exist on the Northeastern area of the study area. When examined on Google Maps, the space appears to be an industrial facility. In Hong Kong, the space examined exist close to the center of the study area. When examined on Google Maps, the space appears to be an railroad juntion. In Olomouc, the space examined exist on the Eastern edge of the study area. When examined on Google Maps, the space appears to be an industrial facility. In all three examples, the edges included in the Official dataset, but not the OSM derived dataset, are access roads that are for private establishments. These edges should not be included in the analysis, and this further proves that the OSM dataset is more robust. 
```

Comment:
Great! Let's change "further proves that the OSM dataset is more robust" to "demonstrates how OSM data can be more appropriate for urban accessibility analytics."